train.py

```diff
@@ -233,14 +239,41 @@ def train(hyp, opt, device, tb_writer=None):
     model.class_weights = labels_to_class_weights(dataset.labels, nc).to(device) * nc  # attach class weights
     model.names = names
 
+    qat = False
+    # SparseML Integration
+    if not opt.use_leaky_relu:  # use LeakyReLU activations
```

Comment:
do we require leaky_relu? should we log/warn?

Reply:
It's the best tradeoff for us between speed and accuracy for pruned_quant.  The default SiLU will not export to ONNX, I'll add a warning when we detect QAT present, and also wrap the ONNX export in a try/except